ansible/README.md

```diff
@@ -25,7 +25,7 @@ brew install python
 pip install ansible==2.0.2.0
 
 cd ansible
-ansible-playbook setup.yml [-e docker_machine_name=whisk]
+ansible-playbook -i environments/mac setup.yml [-e docker_machine_name=whisk]
```

Comment:
@lehoanganh is it an error to run ansible-playbook setup.yml without an environment then? It appears to have the same effect compared to specifying the environment.